### DISCUSSION TITLE
Be able to set an input value programmatically

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#21157] [Feature Requests]
### Summary

For instance, when users create a support request, I want to keep the browser version.
So I set it programmatically. I see it in the input.
But it is not sent to the DB.

### Basic Example

let el_browser = document.querySelector('input[collection="support"][field="browser"]')
if (el_browser) {
	el_browser.value = browserNameAndVersion
}

On a more Vue way,
this.$set(this.formData, 'browser', this.formData.browser);
does not work.

### Motivation

To gather necessary datas that are set programmatically 

### Detailed Design

Inside an extension

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

No

### Alternatives

Being able to set the Default value through a script.

### Adoption Strategy

--

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
